apps/frontend/src/pages/project/libs/components/setup-analytics-modal/setup-analytics-modal.tsx

```diff
@@ -44,6 +49,12 @@ const SetupAnalyticsModal = ({
 		[handleSubmit, dispatch],
 	);
 
+	const handleCopyClick = useCallback(() => {
+		if (hasProjectApiKey) {
+			void dispatch(projectApiKeyActions.copy(project.apiKey as string));
+		}
```

Comment:
let's add isDisabled to icon button instead
```suggestion
		void dispatch(projectApiKeyActions.copy(project.apiKey as string));
```